generators/aws/lib/s3.js

```diff
@@ -107,8 +107,8 @@ var findWarFilename = function findWarFilename(buildFolder, callback) {
             error(err, callback);
         }
         files.filter(function (file) {
-            return file.substr(-FILE_EXTENSION.length) === FILE_EXTENSION;
-        })
+                return file.substr(-FILE_EXTENSION.length) === FILE_EXTENSION;
```

Comment:
Good catch !
Yes the original looks correct.
I don't know why Idea would format it that way.
I'll see if that is a bug in Idea.